server/webapp/WEB-INF/rails/webpack/views/components/page_title/index.js.msx

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
@ketan -- we have one implemented already : https://github.com/gocd/gocd/blob/master/server/webapp/WEB-INF/rails/webpack/views/components/header_panel/index.js.msx